docs/LowerBound_Dist_Profile_Derivation.ipynb

```diff
@@ -0,0 +1,2319 @@
+{
```

Comment:
<p>Fix spelling of "length", "subsequences"</p><p> </p><p>Also, I recommend "between two subsequences starting at locations <code>i</code> and <code>j</code> , respectively, and both of length <code>m</code> is defined as follows"  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
I will fix those and revise the sentence.